master/buildbot/steps/trigger.py

```diff
@@ -167,14 +189,18 @@ def getAllGotRevisions(self):
         return all_got_revisions
 
     @defer.inlineCallbacks
-    def worstStatus(self, overall_results, rclist):
+    def worstStatus(self, overall_results, rclist, unimportant_brids):
         for was_cb, results in rclist:
             if isinstance(results, tuple):
-                results, _ = results
+                results, brids_dict = results
 
             if not was_cb:
                 yield self.addLogWithFailure(results)
                 results = EXCEPTION
+
+            # continue if this brid has to be ignored
+            if len(set(unimportant_brids) & set(brids_dict.values())) != 0:
```

Comment:
what I had in mind: if a scheduler is marked as unimportant, then all the builders this scheduler might kick do not affect the result os the step.